src/components/NumericStepper/__tests__/NumericStepper.spec.tsx

```diff
@@ -163,53 +160,88 @@ describe('<NumericStepper> component', () => {
             <NumericStepper onChange={onChangeMock} minValue={2} maxValue={4} defaultValue={3} />
         );
 
-        await increaseClick();
+        await increaseClick(user);
 
         expect(onChangeMock).toBeCalledTimes(1);
         expect(onChangeMock).toBeCalledWith(4);
         expect(screen.getByRole('spinbutton')).toHaveAttribute('value', '4');
-        expect(onChangeMock).toHaveBeenCalledWith(4);
 
         const inputButton = screen.getByRole('spinbutton');
         // Imagine users enters empty string
         await user.type(inputButton, '   ');
 
-        inputButton.blur();
+        await inputButton.blur();
 
         expect(inputButton).toHaveAttribute('value', '4');
     });
 
     it('should correctly react on increase click when top edge is overstepped', async () => {
+        const user = userEvent.setup();
         view = render(
             <NumericStepper onChange={onChangeMock} step={2} maxValue={4} defaultValue={3} />
         );
 
-        await increaseClick();
+        await increaseClick(user);
 
         expect(onChangeMock).toBeCalledWith(4);
         expect(screen.getByRole('spinbutton')).toHaveAttribute('value', '4');
         expect(onChangeMock).toHaveBeenCalledWith(4);
     });
 
     it('should correctly react on decrease click when bottom edge is overstepped', async () => {
+        const user = userEvent.setup();
         view = render(
             <NumericStepper onChange={onChangeMock} step={2} minValue={3} defaultValue={4} />
         );
 
-        await decreaseClick();
+        await decreaseClick(user);
 
         expect(onChangeMock).toBeCalledWith(3);
         expect(screen.getByRole('spinbutton')).toHaveAttribute('value', '3');
         expect(onChangeMock).toHaveBeenCalledWith(3);
     });
 
-    // it('should react on keyup/keydown press', () => {
-    //     /**
-    //      * TODO: Test with E2E tests.
-    //      * As we actually validate data on blur, its quite challenging to chain together mouseEnter, keyDown and blur.
-    //      * I've tried multiple ways to do it, and don't want to spend way more time on that.
-    //      */
-    // });
+    it('should react on keydown press', async () => {
```

Comment:
normally keydown is a type of event. I think you want to say something like `should decrease value on keyboard interaction`

Reply:
I used original name

Reply:
Sorry, I'm not sure what do you mean. original name of what?

Reply:
should react on keyup/keydown press